ratings/converters.py

```diff
@@ -0,0 +1,8 @@
+class FloatConverter:
+    regex = '[+-]?([0-9]*[.])?[0-9]+'
```

Comment:
Is the plus sign required? I don't see it in the orignal regex or any test failing without it.

Reply:
you are right, I erased it